tests/integration/cmor/_fixes/cmip6/test_cesm2_waccm.py

```diff
@@ -6,48 +6,13 @@
 import iris
 import numpy as np
 import pytest
-from netCDF4 import Dataset
 
 from esmvalcore.cmor._fixes.cmip6.cesm2 import Tas as BaseTas
 from esmvalcore.cmor._fixes.cmip6.cesm2_waccm import Cl, Cli, Clw, Tas
 from esmvalcore.cmor.fix import Fix
 
-
-@pytest.fixture
-def cl_file(tmp_path):
-    """Create netcdf file with similar issues as ``cl``."""
-    nc_path = os.path.join(tmp_path, 'cesm2_waccm_cl.nc')
-    dataset = Dataset(nc_path, mode='w')
-    dataset.createDimension('lev', size=2)
-    dataset.createDimension('bnds', size=2)
-
-    # Dimensional variables
-    dataset.createVariable('lev', np.float64, dimensions=('lev',))
-    dataset.createVariable('lev_bnds', np.float64, dimensions=('lev', 'bnds'))
-    dataset.variables['lev'][:] = [1.0, 2.0]
-    dataset.variables['lev'].bounds = 'lev_bnds'
-    dataset.variables['lev'].units = '1'
-    dataset.variables['lev_bnds'][:] = [[0.5, 1.5], [1.5, 3.0]]
-    dataset.variables['lev_bnds'].standard_name = (
-        'atmosphere_hybrid_sigma_pressure_coordinate')
-    dataset.variables['lev_bnds'].units = '1'
-    dataset.variables['lev_bnds'].formula_terms = (
-        'p0: p0 a: a_bnds b: b_bnds ps: ps')
-
-    # Coordinates for derivation of pressure coordinate
-    dataset.createVariable('a', np.float64, dimensions=('lev',))
-    dataset.createVariable('a_bnds', np.float64, dimensions=('lev', 'bnds'))
-    dataset.createVariable('b', np.float64, dimensions=('lev',))
-    dataset.createVariable('b_bnds', np.float64, dimensions=('lev', 'bnds'))
-    dataset.variables['a'][:] = [1.0, 2.0]
-    dataset.variables['a'].bounds = 'a_bnds'
-    dataset.variables['a_bnds'][:] = [[1.5, 0.0], [3.0, 1.5]]
-    dataset.variables['b'][:] = [0.0, 1.0]
-    dataset.variables['b'].bounds = 'b_bnds'
-    dataset.variables['b_bnds'][:] = [[0.5, -1.0], [2.0, 0.5]]
-
-    dataset.close()
-    return nc_path
+SAMPLE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                           os.pardir, 'sample_data')
```

Comment:
```suggestion
from pathlib import Path

SAMPLE_PATH = Path(__file__).absolute().parents[1] / 'sample_data'
```

Reply:
This statement is at the top of every file. Would it be an idea to create a fixture that defines the sample path? Or some other way to single-source this (e.g. via an import or in conftest.py)?